<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ $TROLLTECH$. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $TROLLTECH_DUAL_LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>qdesigner_internal::FindIconDialog</class>
 <widget class="QDialog" name="qdesigner_internal::FindIconDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find Icon</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QRadioButton" name="m_specify_file_input" >
       <property name="text" >
        <string>Specify image file</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_specify_resource_input" >
       <property name="text" >
        <string>Specify resource</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_widget_stack" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="m_file_input" >
           <property name="editable" >
            <bool>true</bool>
           </property>
           <property name="insertPolicy" >
            <enum>QComboBox::NoInsert</enum>
           </property>
           <property name="autoCompletion" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_cd_up_button" >
           <property name="text" >
            <string>Parent directory</string>
           </property>
           <property name="icon" >
            <iconset/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_icon_view" >
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page" />
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../designer/designer.qrc" />
  <include location="../../components/formeditor/formeditor.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdesigner_internal::FindIconDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>72</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel" >
     <x>91</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdesigner_internal::FindIconDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>287</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel" >
     <x>191</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
